destination/conn.go

```diff
@@ -245,13 +245,13 @@ func (c *Conn) HandleData() {
 // deals with pickle errors internally because retrying wouldn't help anyway
 func (c *Conn) Write(buf []byte) (int, error) {
 	if c.pickle {
-		dp, err := parseDataPoint(buf)
+		dp, err := ParseDataPoint(buf)
```

Comment:
I exported these `parse*()` functions so I could use them from the route package.